feincms_extensions/content_types.py

```diff
@@ -0,0 +1,47 @@
+from feincms.content.medialibrary.models import MediaFileContent
+from feincms.content.richtext.models import RichTextContent
+from feincms.content.section.models import SectionContent
+
+
+class JsonRichTextContent(RichTextContent):
+    class Meta(RichTextContent.Meta):
+        abstract = True
+
+    def json(self, **kwargs):
+        """Return a json serializable dictionary containing the content."""
+        return {'html': self.text}
+
+
+def mediafile_data(mediafile):
+    """Return json serializable data for the mediafile."""
+    if mediafile is None:
+        return None
+    return {
+        'url': mediafile.file.url,
+        'type': mediafile.type,
+        'created': mediafile.created,
+        'copyright': mediafile.copyright,
+        'file_size': mediafile.file_size,
```

Comment:
Media files have related (inline) `MediaFileTranslation` for the `caption` and a m2m `categories` to `Category`.  We rarely use either so they are not essential to get this `shipped` but perhaps could be considered for the future.